Makefile.helper.mk

```diff
@@ -39,7 +39,7 @@ test-e2e-upgrade: go-deploy-manifests
 
 test-e2e:
 	for d in basic; do \
-          KUBERNETES_CONFIG="$(KUBECONFIG)" go test -v -timeout 40m --tags=e2e ./test/e2e/$$d -ginkgo.v -ginkgo.noColor -ginkgo.failFast || exit; \
+          KUBERNETES_CONFIG="$(KUBECONFIG)" go test -v -timeout 40m ./test/e2e/$$d -ginkgo.v -ginkgo.noColor -ginkgo.failFast || exit; \
```

Comment:
Why here we use `go test` while in UT we use `go run` ?

Reply:
For unit tests, we don't have a choice: only the `ginkgo` command supports `-skipPackage`.

Reply:
So why don't we use ginkgo in the e2e tests as well?

Reply:
@qbarrand ?

Reply:
Sorry @ybettan, this thread was marked as resolved so I didn't see your comment. We are not running Ginkgo via `go run` here because we don't need to; there's no flag that we need that would only be available in Ginkgo. Is that Ok with you?

Reply:
Yes, just was wondering what is the difference.